modules/wireguard/README.md

```diff
@@ -0,0 +1,53 @@
+# Wireguard
+
+[`Wireguard`](https://www.wireguard.com/) is an modern open-source VPN that utilizes **state-of-the-art** cryptograph.
+
+This module will monitor one or more vpn network interface and wireguard peers, depending on your configuration.
+
+## Requirements
+
+- `Wireguard` enabled
+- Set network capability to golang collector (default file is: `/usr/libexec/netdata/plugins.d/go.d.plugin`)
+
+E.g:
+```bash
+$ sudo setcap CAP_NET_ADMIN+epi /usr/libexec/netdata/plugins.d/go.d.plugin
```

Comment:
FYI On a systemd system I found that I had to alter systemd unit file and add `CapabilityBoundingSet=CAP_NET_ADMIN` there.
Setting capability alone was not enough and threw errors like `Operation not permitted` in `error.log`

Reply:
You are right, we need to update netdata systemd unit CapabilityBoundingSet.

>  to alter systemd unit file

No need to alter the file.

[(source)](https://www.freedesktop.org/software/systemd/man/systemd.unit.html)

> Along with a unit file foo.service, a "drop-in" directory foo.service.d/ may exist. All files with the suffix ".conf" from this directory will be parsed after the unit file itself is parsed. This is useful to alter or add configuration settings for a unit, without having to modify unit files.


So something like this will do

```cmd
mkdir /etc/systemd/system/netdata.service.d
echo -e '[Service]\nCapabilityBoundingSet=CAP_NET_ADMIN' | tee /etc/systemd/system/netdata.service.d/unset-capability-bounding-set.conf
systemctl daemon-reload
systemctl restart netdata.service
```